pkg/logger/hooks/syslog.go

```diff
@@ -0,0 +1,19 @@
+// +build !windows
+
+package hooks
```

Comment:
[nitpick] "hook" is the logrus term and is not very explicit, renaming this package to "syslog" would make thinks clearer IMHO